### DISCUSSION TITLE
Ability to add range of quantity of ingredients

### DISCUSSION THREAD

**Discussion by TheHYPO35** [mealie-recipes/mealie#3128] [Feature Request]
### First Check

- [X] I used the GitHub search to find a similar requests and didn't find it.
- [X] Checked the [tasks tagged](https://github.com/mealie-recipes/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

### Please provide a concise description of the problem that would be addressed by this feature.

Some recipes include ingredients with ranges of quantities either combined with a unit or alone (such as 2-3 tsp salt or 8-10 shrimp). As far as I can see, there is no elegant way implement this in the ingredient quantity system. It would be nice to be able to somehow include this.

Currently you could create a whole new food list item for "8-10 shrimp", or you could put the quantity in a "note", neither of which is optimal.

Edit: An additional related feature that would also be useful would be a checkbox or other flag that the measurement is approximate. Lots of recipes have "about a cup..." or "around 5..." of something.  This could be denoted in the recipe by the universal "~" tilde for "approximate".

### Please provide a concise description of the feature that would resolve your issue.

I can't say that I have a envisioned the perfect solution to how to implement it, but it would be nice to be able to include quantities that are ranges instead of fixed numbers. Perhaps this needs to be a text field or an option to add a minimum and maximum quantity for a specific ingredient.

### Please consider and list out some caveats or tradeoffs made in your design decision

A text field would not be compatible with the scaling system, but neither is hard coding the range into food name or notes sections. A min-max field could be scaled

### Additional Information

- [ ] If this is accepted I'm willing to submit a PR to provide this feature
- [ ] If this is accepted I'm willing to help maintain this feature
- [ ] I'm willing to sponsor/pay a developer to do this work

**Comment by TheElbinho** [comment]
Duplicate of #2989

**Comment by michael-genson** [comment]
Closing in favor of https://github.com/mealie-recipes/mealie/discussions/2989
